core/templates/pages/contributor-dashboard-page/contributor-stats/contributor-stats.component.html

```diff
@@ -0,0 +1,604 @@
+<div class="stats-container">
+  <div class="stats-contribution-type-mobile-container">
+    <div class="dropdown-label">Type</div>
+    <div #mobileDropdown class="oppia-contribution-stats-type-selector-mobile-container e2e-test-topic-selector">
+      <div class="oppia-stats-type-selector-inner-container e2e-test-topic-selector-selected" (click)="toggleMobileDropdown()">
+        {{selectedContributionType}}
+        <span class="material-icons oppia-contributions-dropdown-arrow" [ngStyle]="{ 'transform': mobileDropdownShown ? 'scaleY(-1)' : 'scaleY(1)' }">
+          arrow_drop_down
+        </span>
+      </div>
+
+      <div *ngIf="mobileDropdownShown" class="oppia-stats-type-selector-dropdown-container e2e-test-topic-selector-dropdown">
+        <div class="oppia-stats-type-selector-dropdown-option e2e-test-topic-selector-option"
+             (click)="selectOption(option.contributionType)"
+             *ngFor="let option of options"
+             [ngClass]="{'oppia-stats-type-selector-dropdown-option-selected' : option === activeTopicName}">
+            {{option.displayName}}
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="stats-contribution-type-label">
+    Contribution Type
+  </div>
+  <div class="stats-welcome-text-container">
+    Explore the impact of your contributions!
+  </div>
+  <div class="stats-contribution-type-container">
+    <div #dropdown class="oppia-stats-type-selector-container e2e-test-topic-selector">
+      <div class="oppia-stats-type-selector-inner-container e2e-test-topic-selector-selected" (click)="toggleDropdown()">
+        {{selectedContributionType}}
+        <span class="fas fa-caret-down oppia-stats-type-selector-inner-container-arrow" [ngStyle]="{ 'transform': dropdownShown ? 'scaleY(-1)' : 'scaleY(1)' }">
+        </span>
+      </div>
+
+      <div *ngIf="dropdownShown" class="oppia-stats-type-selector-dropdown-container e2e-test-topic-selector-dropdown">
+        <div class="oppia-stats-type-selector-dropdown-option e2e-test-topic-selector-option"
+             *ngFor="let option of options"
+             (click)="selectOption(option.contributionType)"
+             [ngClass]="{'oppia-stats-type-selector-dropdown-option-selected' : option === activeTopicName}">
+          {{option.displayName}}
+        </div>
+      </div>
+    </div>
+  </div>
+  <div *ngIf="type == 'translationContribution' || type == 'translationReview'" class="stats-cards-container">
+    <mat-card *ngFor="let page of statsData[type] | keyvalue" layout="row" class="list-view-item">
```

Comment:
It doesn't look like we ever use `page.key`, so it seems simpler to just iterate over statsData[type].values() in the first place. Also let's update the name since it's not actually the page we're iterating over, but the stat object itself. Ditto below.

```suggestion
    <mat-card *ngFor="let translationStatByLanguageAndTopic of statsData[type].values()" layout="row" class="list-view-item">
```

Reply:
Keeping keyvalue as it is since we are removing language from `PageableStats`